ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -3875,4 +3944,20 @@ CSWTC:
   - OPERATIONAL
   uses:
   - condensing_supply_water_temperature_setpoint
-  - condensing_supply_water_temperature_sensor
\ No newline at end of file
+  - condensing_supply_water_temperature_sensor
+
+HTWHLSPC:
```

Comment:
HTWHLSPC is an 'alternative' to ECON/mixed air unit for AHUs. These are some of the points that would be expected from a heatwheel AHU.